### PR TITLE
Fix/user pagination

### PR DESCRIPTION
**Pull request by neb42**
Jira ticket: JIRA-CORE24-#

## What have you implemented?
* Move pagination to models lib
* Fix pagination parsing
* Parse me before sending payload
* Seed users

### CODE REVIEW

File: libs/db/seeds/create_users.ts [hunk 1]
```diff
@@ -9,22 +9,60 @@ export async function seed(knex: Knex): Promise<void> {
...
+      oktaId: '00u7gyomqtlFFSaNW5d7',
+      userName: 'irena.lammerer@nrc.no',
+      firstName: 'Irena',
+      lastName: 'Lammerer',
+      displayName: 'Irena Lammerer',
+      emails: JSON.stringify([
+        {
+          primary: true,
+          value: 'irena.lammerer@nrc.no',
+          type: 'work',
```
Comment (internetti) [inline libs/db/seeds/create_users.ts:24]: what is this meant for, what other types are you thinking of?

**Review by internetti — APPROVED** [review]

**Review by internetti — CHANGES_REQUESTED** [review]
rebased the user info on this, getting this from the me endpoint now

ZodError: [
  {
    "code": "invalid_type",
    "expected": "date",
    "received": "string",
    "path": [
      "createdAt"
    ],
    "message": "Expected date, received string"
  },
  {
    "code": "invalid_type",
    "expected": "date",
    "received": "string",
    "path": [
      "updatedAt"
    ],
    "message": "Expected date, received string"
  }
]

File: libs/db/seeds/create_users.ts (hunk 1 above)
Comment (neb42) [reply libs/db/seeds/create_users.ts:24]: This comes from okta. I'd be happy to change the format of emails in our user db, but out of scope for this pr.

Comment (internetti) [reply libs/db/seeds/create_users.ts:24]: yes yes, this question was just out of curiosity

**Review by internetti — APPROVED** [review]

**Pull request merged**